### PR TITLE
Draft documentation for About in Kor.

### PR DESCRIPTION
**Pull request by MoonkiHong**
This is the draft for the document about "About" section in Korean. (Repost of #6)

### CODE REVIEW

**Review by fesfojas — APPROVED** [review]
No additional comments. Looks good :)

**Review by limisaac — APPROVED** [review]
Lets merge!

File: _i18n/ko/about/about.md [hunk 1]
```diff
@@ -1,44 +1,32 @@
...
-[Template README]: https://github.com/just-the-docs/just-the-docs-template/blob/main/README.md
-[use the template]: https://github.com/just-the-docs/just-the-docs-template/generate
\ No newline at end of file
+# About : 오소리 프로젝트 소개
+
+- 명칭
+    - 오소리(OSORI;Open Source DB Integration)는 “오픈소스 소리내다!” 라는 의미를 담고 있습니다. 자발적인 기여와 협력을 통해 더 나은 오픈소스 생태계를 만들겠다는 의지를 표현하였습니다.
+- 목적
+    - 오소리 프로젝트는 오픈소스 정보 데이터를 통합∙공개하여 누구나 쉽게 이를 확인할 수 있게 함으로써 보다 투명하고 신뢰성 있는 오픈소스 생태계 구축에 기여하는 것을 목적으로 합니다.
+
```
Comment (MoonkiHong) [inline _i18n/ko/about/about.md:7]:
```suggestion

```

Let us remove this line.

**Review by MoonkiHong — COMMENTED** [review]
PTAL. 🙏

**Pull request merged**